src/command.rs

```diff
@@ -223,6 +226,7 @@ impl Command {
                 pack!(buf, 0x01, [lower, upper, scanning_seq_and_dir])
             }
             GateDrivingVoltage(voltages) => pack!(buf, 0x03, [voltages]),
+            SourceDrivingVoltageYellow(vsh) => pack!(buf, 0x04, [vsh]),
```

Comment:
I guess my question here, is what happens if i set `SourceDrivingVoltage(vsh1, 0, 0)`? I think when I was porting from the python inky driver, they only sourced the driving voltage to yellows with a packed version with 0x04, 0x07.